src/mobu/handlers/github_refresh_app.py

```diff
@@ -32,28 +32,30 @@
 )
 async def post_webhook(
     context: Annotated[RequestContext, Depends(anonymous_context_dependency)],
-    refresh_app_config: Annotated[
-        GitHubRefreshAppConfig,
-        Depends(github_refresh_app_config_dependency),
+    config: Annotated[
+        Configuration,
+        Depends(config_dependency),
```

Comment:
Nit: If you delete the trailing comma here, I think it will collapse back down to one line.